native/source/mixpanel/detail/worker.cpp

```diff
@@ -190,6 +196,50 @@ namespace mixpanel
             }
         }
 
+        int Worker::parse_www_retry_after(nanowww::Response& response)
+        {
+            auto success = false;
+            static auto failure_count = 0;
```

Comment:
This static variable is global to the program; if you want it to be unique per-`Worker` (and thus thread-safe), you need it to be a member variable of `Worker`.